lintreview/review.py

```diff
@@ -197,10 +197,13 @@ def publish_status(self, problem_count):
             self.publish_ok_comment()
             state = 'success'
             description = 'No lint errors found.'
-        self._repo.create_status(
-            self._pr.head,
-            state,
-            description)
+        status = self.config.get('PULLREQUEST_STATUS', True)
+        if status:
+            self._repo.create_status(
+                self._pr.head,
+                state,
+                description
+            )
```

Comment:
I added that in https://github.com/markstory/lint-review/pull/84/commits/38a6d94a587e67e58e1562805a9ccc2abdedd9a7
I piggy-backed off 2 other tests, that test the label and comment. Let me know if that's fine, otherwise I can refactor.